testing/vcs/test_vcs_lambert.py

```diff
@@ -0,0 +1,31 @@
+import vcs,cdms2
```

Comment:
I might be wrong here but don't we have a generic python script that accepts the projection type as argument and CMake adds a test per projection type?